packs/aws/actions/lib/ec2parsers.py

```diff
@@ -35,6 +130,8 @@ def selector(self, output):
             return self.parseInstance(output)
         elif isinstance(output, boto.ec2.volume.Volume):
             return self.parseVolume(output)
+        elif isinstance(output, boto.ec2.blockdevicemapping.BlockDeviceType):
```

Comment:
since BlockDeviceType is an object(https://github.com/boto/boto/blob/d9e5cfe900e1a58717e393c76a6e3580305f217a/boto/ec2/blockdevicemapping.py#L25) can we use some generic mechanism to parse all python objects.

Writing out FieldList like https://github.com/StackStorm/st2contrib/pull/386/files#diff-417c109ec13ece08b2872bd616c2e1f3R17 leads to too much special casing unfortunately.